connectors/src/connectors/microsoft/lib/cli.ts

```diff
@@ -1,6 +1,6 @@
 import type {
   AdminSuccessResponseType,
-  MicrosoftCheckFileResponseType,
+  CustomDataResponseType,
```

Comment:
IMHO, this type's name isn't explicit enough.
```suggestion
  CheckFileGenericResponseType,
```